microshift_getting_started/microshift-architecture.adoc

```diff
@@ -19,13 +36,30 @@ Learn the specifics of {product-title} architecture including design intent, how
 * Compatible with RPM-OStree-based systems
 * Uses only the APIs needed for essential functions, such as security and runtime controls
 * Enables a subset of commands from the OpenShift CLI (`oc`) tool
+* Does not support high availability (HA) by the adding of worker nodes
+
+.{product-title} differences from {oke}.
+image::microshift-RHDE-v-OpenShiftKE.png[<{product-title} is tasked with only the Kubernetes cluster services networking, ingress, storage, helm, with additional Kubernetes functions of orchestration and security.>]
+
+Figure 2 shows that {oke} performs many more cluster services, such as:
```

Comment:
https://www.redhat.com/en/technologies/cloud-computing/openshift/kubernetes-engine implies that although an OKE deployment may include a bunch of these features, they aren't supported with an OKE subscription.

Reply:
@DanielFroehlich Would it be more accurate to show this as a comparison with OCP?

Reply:
@DanielFroehlich can you weigh in? We went over in the graphics meeting, but can you recap here for @dhellmann?